api/python/notebooks/analysis_demo/comp_bio_geneformer_prediction.ipynb

```diff
@@ -0,0 +1,1003 @@
p1I5MnT8dgkKyYQgjxZSBBkRBCDAHv1a5i6f7XADjQnlq8dUxFKKXoasyosHyGkz50HUskkVLbMWs2+ddej6J0H5mZmcX4DRt4aX46jZkGjDGdonYjpXclUm4rxsRXzsaNH7LZZcM9cRwjSkcy8qxzUHslLlix4h3q6hJBW1tbCxaLlQkTJnPthCt4ZPtTeMNepudM5oTCgYu59pSRkclXvvI1YrEoJpN50Lbt7T4YsAxtQs/7FkII8cUnQZEQQgwBFe3VA+7rtKbWmYlm9z/6YonomGM6YbOZ6PZtGB59GNdxx2MbOSrZpm5yIZe8Xkl7mgHQ2XFeogbZoYCoubkxkfBAUfB63KzzNlPY6SfdlY5v+dtEGupJGz+R1tbmlGsfel3iKuL2Obce9f1DIpA5XEAEsGbNSmKxaK+tiWLUh0ybNktGiYQQ4ktEgiIhhBgChrmKWVO/LvnaZXbSHukAoHH+WJRIM3pFFarNjn7q8RBf1eccYYvK28ePxRN1YwqHmbR6Fb73V1L8o59gG5FY1zP1ouv4l/tPGH2d+ArdnDJqBluat1PuGYXZYCIUCvY9bzhE01Mv4V32JgDeN9+g5Pj57O4xGJOTk9vnuE9LKBToZ6uOqqqcdNIZpKU5cLuPbNqeEEKILwYJioQQYgg4oXAugWiQbS07yLJlcuGoc/GGfaiKSoEjj8aqx/FWVKEFA6Q9+RIXf+s81kT2UemvTZlJphhMEIWoxYLXk052YzP+DeuTQVGxs5DvnHsHrSEvLx94gyd3PQdAibOQm6fdQE5OPk6ni46OdgDS0z1kZmZTue6DlP5mrFjJLIuFppkz8EyfQXn5+CO6z2g0SmPjQcxmK9nZOR/pvSotHUVb25o+2zVNw+PJxP4p1lISQgjx+ZCgSAghhgBFUTir9BTOKj2FlmAr/9zxL3yRDqbnTKbAkUf7iveSbfVYjDFbW1hwxfd4o/IdXtj3Cjo6eV476UFbMkjaXWggopvI7jVqYjVaURWVDU1bktuqOmrZ3LydGblTOOusxezZswtFURg1agxGoxGjJ4O4z9d9El1HDYXI/+BDyr521RGt4QmHw7z66gv4fF4Axo+fzPTps4/qfaqpqSIejzFhwhSqqyvx+bqTOmRl5WCz2Y7qfEIIIb4YJCgSQogh5oHNj1DXWQ9AdUctHqub9N6NDIl1RoFYkCnb/czYEcAe1vFm+Ng5fgwNDh8VRTrrxngomDyM3pPJVEXtfUZURcX7zjLa3ngNl8VKzlcvw2JJpPXPu+ab1P3lPqINDaB3r93RQiG0YLBPIob+7N+/OxkQAWzbtomJE6dgNlsGPqiH3bu3s3r1iuTruXNPABKBktvtYcKEyZJgQQghvqQkKBJCiCEkrsWp66zHHNHIbovRkWagZftGXKFQSjtFNaDpGqt2vc01GzuTM+jcrW344zup8CSClILGCBUrX2fEyXkYnd2VjnLsWSwoOo7lNSsBKPeMZFSbiYOPPZpsU/1/97DrpBPQTSamTZtN9m3/RdX6DzE++QREo0TMJiqnTGb7slfIzy8kJycPk8lMYWHxAMHJkW7r3969u1Ne79u3mzPPPJdRo8Yc8TmEEEJ8MUlQJIQQQ4hBNTBWz2HuS9txBjXiCkQz1vVp917jB9grcjBrap+wIk21ABrzNviZuSMAfEjle3sp+entmDIyku0uHr2Y4wpmEYlHGeYqov3d5aknCoUINjURsttYvvwNDAYDkUgE+8SxlIWiHMzLpSUWgXYf7e0+du3aDsDw4SM44YRT+vR5xIjR7N27i7a2FgAmTZqG2Xz4bHOH9C7oOliBVyGEEF8ufec3CCGE+FJbXOvBGUzUGzLoYG1JZKE7NGnN61BZPULlxQOvM2n0PPaUdE8/M+bkMP/0qxnjzWfGju4sbXGvl/YV7/a5VqEjn9L0ElRFxTZyVDI1N0DIaiFsTZw7Ho8TiUQACDgcbM3y4DekfkVZgonMdRUV+5KJGnoym82cddZiTj99Eeec8xWmTBm8QGtvM2fOw+VKTCR0udKZMePIaiAJIYT44pORIiGEGGLMipG+ibEhYFF4+fh0mjxGoqZEQOK2pjP+5p+zf9MKihQ3RVPnUd1YT3qHHV1RUHqs/1GMRuLxOJ2dfmw2O6Ze64AshUUUfu8WvO8sQzMY2GwEXe3/2ZyiKKSnu2lsrE9uK6iuoy0rA2+GB7XHcfF4PFkzyGg0kpdX8JHeF6fTxXnnXUIkEj7idUhCCCG+HCQoEkKIIcZz2un4131I3N+Rst0e1tEKconGEhnXVEUlw+Lm/zY9SEfEj0k1cW2wBCUQAUWhYkQppXv3owCmwkIMM2bwwgtP4/d3YLFYOL5oBNrqVShGA5nnXUA0I4N39u7A57SRn1+Ae/9+IppGzNw3iYKu6zTX1ZDX0EjIZiO9tY3c+gYiFjPFC04mLc3BwYO1LFv2GvF4DIPBwPz5JzNsWOnHfn8kIBJCiKFH0fUej/k+R/G4Rmtr5+fdDSGEGBJiPh+hA/uJdXSw78WniIfDbBhjo3ZSEb6wj5geJ9PqwWFyUNlRnTyuIC2P7479Oi+++B9isSjmUIgcRzoLLvwq769ZwYEDewGwBoJMWr8JVUtM01MdDvaffCL13tbkufJr6mjIz0XrGuXpz+QP1mMLdieBMI0aRayyEmNGBptKCmjvse5HURQuvvgKLBZZC/RZychIw2CQmfhCiC8+GSkSQoghyJiejmPKVCrbq3ko6gAciR2h7qClJdRGS6gt5bi4ruFyuTnrrMXs2red9nArZTELkcpKotFosl2avzMZEAFofj/RxkYwd3/thC0WPC1ttORkDdjPnRPGMb09gDkex5juxt9V5DVaX0+Z18vGWdOSbXVdx+/3DxgUHTiwl9raKhwOFxMnTk1OuRNCCCEkKBJCiCHMpB6+/s8hqqKycHgi61vcqrOk8y188U6MMZ2F//ShjelOTBBIs6OrKkpXYBQzGOjoNaDgaWkhs6kFd2
```

Comment:
We did switch to `emb_layer=0` right?

Reply:
good catch! No wonder why the data projection did not look good anymore